### PR TITLE
Added Directory on Paper One_shot_Face_Reenactment

### LINKED ISSUE #21: Claim:- One-shot Face Reenactment
Paper :- https://arxiv.org/pdf/1908.03251.pdf

### PR DESCRIPTION
**Pull request by Bharat-mtr**
This is regarding Issue  #21 
Added directory for Paper One_shot_Face_Reenactment.

Paper :- https://arxiv.org/pdf/1908.03251

Fixes #21 

Repository:- https://dagshub.com/Bharat-mtr/One_Shot_Face_Reenactment

### CODE REVIEW

**Comment by Bharat-mtr** [comment]
@nirbarazida  Please review and let me know if I need to add/edit anything.

**Comment by Bharat-mtr** [comment]
Fixes #21

**Review by deanp70 — CHANGES_REQUESTED** [review]
Overall this is another great contribution! Thanks for taking part!

I have one comment to fix – there are a lot of redundant mac related files and folders, mainly a `_MACOSX` folder here: https://dagshub.com/Bharat-mtr/One_Shot_Face_Reenactment/src/master/sample_images/LOFR%20synthetic%20images as well as `.DS_Store` files all over the project. Could you please remove them, as they aren't necessary, take up space, and make navigation more confusing.

After they are removed, we can merge this in.

Thanks again,
Dean

File: One-shot Face Reenactment/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# One-shot Face Reenactment
+
+[[Project]](https://wywu.github.io/projects/ReenactGAN/OneShotReenact.html) [[Paper]](https://arxiv.org/abs/1908.03251) [[Demo]](https://www.youtube.com/watch?v=FE-D6wh11_A)  
+
+Official test script for 2019 BMVC spotlight paper 'One-shot Face Reenactment' in PyTorch.
+
+[DAGsHub Repository](https://dagshub.com/Bharat-mtr/One_Shot_Face_Reenactment)
+
+<img src="https://github.com/bj80heyue/Learning_One_Shot_Face_Reenactment/blob/master/pics/main.png" width = 900 align=middle>
```
Comment (deanp70) [inline One-shot Face Reenactment/README.md:9]:
```suggestion
<img src="https://dagshub.com/Bharat-mtr/One_Shot_Face_Reenactment/raw/master/pics/main.png" width = 900 align=middle>
```

**Comment by Bharat-mtr** [comment]
> Overall this is another great contribution! Thanks for taking part!
> 
> I have one comment to fix – there are a lot of redundant mac related files and folders, mainly a `_MACOSX` folder here: https://dagshub.com/Bharat-mtr/One_Shot_Face_Reenactment/src/master/sample_images/LOFR%20synthetic%20images as well as `.DS_Store` files all over the project. Could you please remove them, as they aren't necessary, take up space, and make navigation more confusing.
> 
> After they are removed, we can merge this in.
> 
> Thanks again, Dean

I have removed the redundancy. Although the DagsHub repo is taking time to get in sync with Github repo changes.

**Comment by deanp70** [comment]
Have you pushed the new git version? It shows me last update was a week ago.

Let me know once it's pushed and I'll re-check

**Comment by Bharat-mtr** [comment]
> Have you pushed the new git version? It shows me last update was a week ago.
> 
> Let me know once it's pushed and I'll re-check

I have pushed changes to corresponding git repo of One_Shot DagsHub Repo 
Git repo :- https://github.com/Bharat-mtr/One_Shot_Face_Reenactment

Its taking time to show same changes in DagsHub Repo

https://dagshub.com/Bharat-mtr/One_Shot_Face_Reenactment

its showing sync in progress, although I have committed the changes in git repo.

**Review by deanp70 — APPROVED** [review]
Looks good to me, the sync has been complete. Will merge this in.

Thanks again for your contribution!

**Pull request merged**